lnwallet/interface_test.go

```diff
@@ -384,6 +407,12 @@ func testDualFundingReservationWorkflow(miner *rpctest.Harness,
 
 	// Mine a single block, the funding transaction should be included
 	// within this block.
+	switch alice.BackEnd() {
```

Comment:
Ahh I see it's usage now. Seems like we could just perform a type switch everywhere this is currently used?